kyuubi-rest-client/src/main/java/org/apache/kyuubi/client/HttpClientFactory.java

```diff
@@ -35,23 +42,35 @@ public class HttpClientFactory {
   public static CloseableHttpClient createHttpClient(RestClientConf conf) {
     RequestConfig requestConfig =
         RequestConfig.custom()
-            .setSocketTimeout(conf.getSocketTimeout())
-            .setConnectTimeout(conf.getConnectTimeout())
```

Comment:
note: `setConnectTimeout` is removed in `httpclient5`, and changed to use ConnectionConfig to set connect timeout.
```
ConnectionConfig.custom()
            .setConnectTimeout(conf.getConnectTimeout(), TimeUnit.MILLISECONDS)
```